### ISSUE TITLE
LogicalKeyboardKey

### ISSUE THREAD

**Issue by bernaferrari** [flutter/flutter#83733]
## Steps to Reproduce

1. ...
2. ...
3. ...

## Logs

<details>
<summary>Logs</summary>

```
```

```
```

</details>

**Comment by HansMuller** [comment]
CC @dkwingsmt

**Comment by dkwingsmt** [comment]
I agree, I also think the `if (Platform.macOS) metaLeft else controlLeft` pattern is too much boilerplate. Let me see if we can come up with a utility class.

**Comment by bernaferrari** [comment]
Hey @dkwingsmt, that code wouldn't work because there is no dart:IO in Flutter Web, so it would be [even more complicated](https://stackoverflow.com/a/67644049).

**Comment by csells** [comment]
cc @cbracken to consider for the desktop stable release

**Comment by dkwingsmt** [comment]
Yeah I've been considering this for a while. It's easy to implement, and for sure important. Just hard to name.

Let's discuss naming. We need a name for the activator class, one for the "control but meta for macOS" field, and one for the "meta but control for macOS" field. My latest thought is 
```
OsSingleActivator(
  this.osControl,
  this.osMeta,
...);
```
Or 
```
AdaptiveSingleActivator(
  this.primary,
  this.auxiliary,
...);
```

**Comment by bernaferrari** [comment]
I prefer the first one. I like the idea of `controlCommand` or `macOSCommand` but they don't work well with meta (metaControl would be confusing 🙁) so I prefer your suggestions `osControl`/`osMeta`.

**Comment by justinmc** [comment]
Currently we maintain separate maps for our default text editing shortcuts.  [Here is Mac's meta+c](https://github.com/flutter/flutter/blob/ab51a0260d29d50800ae5c9665ecb01883a9b503/packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart#L409) and [here is Windows control+c](https://github.com/flutter/flutter/blob/ab51a0260d29d50800ae5c9665ecb01883a9b503/packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart#L460).  There is no slicker way right now that I'm aware of.  If this issue gets fixed maybe we can simplify those maps.

**Comment by bernaferrari** [comment]
Idea 2: we could somehow unify 'delete' with 'backspace' as they are the same shortcut 98% of the time.

**Comment by Peetee06** [comment]
I face this issue just now and would really like an easier interface. How about "platformControl" and "platformMeta"?

**Comment by AlabasterAxe** [comment]
I think this issue should be renamed to something like "Create a utility to create shortcuts that use cmd on Mac and ctrl on Windows/Linux". The title as written is not a bug; `ctrl+c` should not be handled as `cmd+c` on a Mac by default.

FWIW, Electron has a "CommandOrControl" activator for their shortcuts system which does exactly what's described here: https://www.electronjs.org/docs/latest/api/accelerator

A little wordy for my tastes but it gets the point across.
